src/store/applicationSettings.js

```diff
@@ -27,6 +27,6 @@ export const actions = {
 
 export const getters = {
   getPathManagementStrategy({ pathManagementStrategy }) {
-    return pathManagementStrategy !== PathManagementStrategy.NotSet ? pathManagementStrategy : PathManagementStrategy.RcFiles;
+    return pathManagementStrategy;
```

Comment:
Changing this getter will remove the default that we provide in `src/pages/PathUpdate.vue`. I think we have two choices to resolve this:

1. Revert this change and update `src/pages/FirstRun.vue` so that we don't write the `pathManagementStrategy` in the `onChange` method
2. Update `src/pages/PathUpdate.vue` so that it will properly set a default value

Reply:
Nice catch. I opted for 2 - it should work properly now.